app/routers/router.ts

```diff
@@ -251,6 +252,9 @@ export default function router({ skipEarthdataLogin = 'false' }: RouterConfig):
 
   result.get('/admin/configuration/log-level', asyncHandler(setLogLevel));
 
+  result.get('/capabilities', asyncHandler(getCollectionCapabilitiesJson));
+  // Enable HTML view with HARMONY-1393
+  // result.get('/capabilities.html', asyncHandler(getCollectionCapabilitiesHtml));
```

Comment:
I thought you were planning to use the Accept header to determine what type to use for the response?